core/Tracker.php

```diff
@@ -355,4 +363,13 @@ private static function isDebugEnabled()
 
         return false;
     }
+
+    public function isPreFightCorsRequest()
```

Comment:
for new methods we should already try to add typehints where possible
```suggestion
    public function isPreFightCorsRequest(): bool
```